# Header
std_msgs/Header header

# Position and description
atak_bridge/PoseDescription[] pose_list

